src/web/urls.py

```diff
@@ -11,4 +11,6 @@
     path('groups/<int:pk>', views.DetailTechGroup.as_view(), name="get_tech_group"),
     path('events/<int:pk>', views.DetailEvent.as_view(), name="get_event"),   
     path('build_sidebar', views.BuildSidebar.as_view(), name="build_sidebar"),
+    path("get_event_details/<int:pk>", views.GetEventDetailsModal.as_view(), name="get_event_details"),
```

Comment:
I think this URL should be nested under `events/...`.

Reply:
are you thinking like `events/<int:pk>/details` ?

Reply:
Yeah!